frontend-react/src/pages/Upload.tsx

```diff
@@ -348,7 +348,7 @@ export const Upload = () => {
                         id="upload-csv-input"
                         name="upload-csv-input"
                         aria-describedby="upload-csv-input-label"
-                        accept=".csv, text/csv"
+                        accept="text/csv, .csv"
```

Comment:
What a weird bug. Did it prioritize the extension over the mime type because it was the first of the two values? I figured a capitalized extension would have it fall back on the mime type. Great catch!

Reply:
agreed, and that was my though process: "what if it is evaluating in order...? :thinking:" :laughing:

Reply:
https://github.com/trussworks/react-uswds/blob/f795114496d25865894e71e68d40a85a7ccb971d/src/components/forms/FileInput/FileInput.tsx#L96-L122

Here's the function itself! Looks like it was looking at the name and filetype to find `.csv` in lowercase, couldn't find it, but the statement `file.name.indexOf(fileType) > 0 || file.type.includes(fileType.replace(/\*/g, ''))` is wrapped in an iterator that goes through each `fileType` given in the `accept` array (split on the comma)...it's super weird that if the lowercase `.csv` was false and false, that it'd _not_ continue iterating and find `text/csv` with that same statement on the next iteration.